src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
+  (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+  )
+);
```

Comment:
You don't need additional round brackets if you don't write word `return ` 
```suggestion
}) => (
  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
);
```